### DISCUSSION TITLE
How to cast the entire mp3-playlist to Chromecast (Windows 11)

### DISCUSSION THREAD

**Discussion by Scorrrp** [smplayer-dev/smplayer#866] [General]
I´m a new user of SMPlayer. I installed SMPlayer to my Windows 11 yesterday, created a mp3-playlist and tried to play it. SMPlayer connects to my Brave browser, plays one song and stops. It does not go to the next track of playlist. Could somebody help the novice? 
Regards, Scorrrp

**Comment by smplayer-dev** [comment]
I'm afraid this is not implemented.

**Comment by Scorrrp** [comment]
Thanks. You can create a playlist to SMPlayer. What is it for if you can cast only one mp3 to Chromecast instead the entire playlist?
